bucket/virt-viewer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "10.0-1.0",
+    "description": "Open source tool to connect to virtual machines to display VNC or SPICE",
+    "homepage": "https://virt-manager.org/download/",
+    "license": "GPL v3+",
```

Comment:
Invalid SPDX identifier

Reply:
I will change it to SPDX format "GPL-3.0-or-later"